layouts/includes/announcements.html

```diff
@@ -0,0 +1,9 @@
+{% if conference_data.slug == 'berlin' %}
+    <section id="announcements" class="section section-night">
+        <div class="section_body">
+            <h2>Announcing Ignite Berlin!</h2>
+            <p>We're excited to host an <a href="http://igniteberlin.com/ignite-at-viewsource/ ">Ignite Berlin Special</a> on 12 September! Your View Source ticket includes admission to Ignite; non-View Source tickets to Ignite are €10. This is sure to be the largest Ignite Berlin ever!</p>
```

Comment:
I was confused by the copy. Is there a special on tickets for Ignite Berlin? Is there a special on View Source tickets if I buy an Ignite ticket? Is it a Special Ignite event? Do I have to buy my ticket on the 12th?